### PR TITLE
Instructions for finding more PRs to approve

### PR DESCRIPTION
**Pull request by elharo**
@suztomo

### CODE REVIEW

File: boms/cloud-oss-bom/RELEASING.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ new release is available on Maven Central.
     * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
 * Merge the dependabot PR that updates libraries-bom in https://github.com/GoogleCloudPlatform/java-docs-samples/pulls
```
Comment (suztomo) [inline boms/cloud-oss-bom/RELEASING.md:35]:
```suggestion
* Merge the RenovateBot PR that updates libraries-bom in https://github.com/GoogleCloudPlatform/java-docs-samples/pulls
```

**Review by suztomo — APPROVED** [review]
I think I forgot to check the renovate bot PRs.

File: boms/cloud-oss-bom/RELEASING.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ new release is available on Maven Central.
...
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
+    * Go to go/java-live
+    * Sort by title
+    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v<version>"
```
Comment (suztomo) [inline boms/cloud-oss-bom/RELEASING.md:39]: I see 104 PRs as of now...

Comment (elharo) [reply boms/cloud-oss-bom/RELEASING.md:39]: Jeff and Stephanie are working on some ways to automate more of this.

**Pull request merged**